simpleclient/src/main/java/io/prometheus/client/Gauge.java

```diff
@@ -181,6 +181,21 @@ public void setToCurrentTime() {
     public Timer startTimer() {
       return new Timer(this);
     }
+
+    /**
+     * Executes runnable code (ie Java 8 Lambda) and observes a duration of how long it took to run.
+     *
+     * @param timeable Code that is being timed
+     */
+    public void time(Runnable timeable){
```

Comment:
The important thing about those functions is the noLabelsChild. Each of those mirrors the methods on a Child, so you need to add a single `time` function here.